source/docs/software/driverstation/using-the-labview-dashboard-with-c++-java-code.rst

```diff
@@ -0,0 +1,110 @@
+Using the LabVIEW Dashboard with C++\Java Code
+==============================================
+
+The default LabVIEW Dashboard utilizes Network Tables to pass values and is therefore compatible with C++ and Java robot programs. This article covers the keys and value ranges to use to work with the Dashboard.
+
+Drive Tab
+---------
+
+.. image:: images/using-the-labview-dashboard-with-c++-java-code/drive-tab.png
+
+Most of the indicators on the Drive tab utilize arrays. The c++ and Java SmartDashboard classes do not currently support sending arrays.
```

Comment:
The C++ and Java classes now support arrays, so this comment can be deleted

Reply:
Removed in 6e8cea0.